pkg/informer/delegating_event_handler.go

```diff
@@ -121,3 +123,19 @@ func (d *delegatingEventHandler) getEventHandler(resource schema.GroupResource,
 
 	return groupResourceHandlers[clusterName]
 }
+
+func clusterNameForObj(obj interface{}) logicalcluster.Name {
```

Comment:
Could this function be replaced with called to https://github.com/kcp-dev/logicalcluster/blob/main/name.go#L93?

Reply:
I think in some particular cases, like offline deletion, the object is a `cache.DeletedFinalStateUnknown`, in which case the `logicalcluster.Name` function cannot be called.

Reply:
Got it, thanks!